content/install_maintain/installation/certificates.md

```diff
@@ -0,0 +1,79 @@
+A client connecting to that server will perform the certification path validation algorithm:
+
+1. The subject of the certificate matches the hostname (i.e. domain name) to which the client is trying to connect;
+1. The certificate is signed by a trusted certificate authority (CA).
+  
+(Taken from [Wikipedia](https://en.wikipedia.org/wiki/Public_key_certificate))
+
+
+## Cluster certificates setup
+The Cloudify Manager cluster uses the TLS protocol (as described above) for:
+
+1. Communication between the PostgreSQL cluster nodes.
+1. Communication between the RabbitMQ cluster nodes.
+1. Communication between the Cloudify Management service cluster nodes and the other services.
+
+A few notes: 
+
+* The certificates/keys should be created before proceeding with the installation process and in a PEM format.  
+* The certificates/keys are copied to `/etc/cloudify/ssl` during installation from the source given by the user. 
+Therefore, it is up to the user to delete the leftovers from the source location.  
+* In case of using externally hosted PostgreSQL or RabbitMQ instances, the CA needs to be 
+retrieved from the cloud service hosting the instance.
+* The **same CA** should be used to sign all cluster hosts' certificates.  
+
+The following files should exist on each host:
+
+1. CA certificate - The CA certificate that signed the hosts' public key certificates. 
+1. Public key certificate - A public key certificate signed by the given CA that specifies the host IP and username.
+1. Private key - The private key associated with the host's public key certificate.
+  
+The `cfy_manager generate-test-cert` command can be used for creating example certificates:
+
+On a host that has Cloudify installed, generate certificates for all hosts using:
+```
+cfy_manager generate-test-cert -s <manager 1 ip>,<manager 1 hostname>
+cfy_manager generate-test-cert -s <manager 2 ip>,<manager 2 hostname>
+cfy_manager generate-test-cert -s <manager 3 ip>,<manager 3 hostname>
+cfy_manager generate-test-cert -s <postgres server 1 ip>,<postgres 1 hostname> 
+..
+..
+cfy_manager generate-test-cert -s <rabbitmq server 3 ip>,<rabbitmq server 3 hostname> 
+```  
+
+{{% note %}}  
+The commands above should be run on the same host.
+For production purposes, please use a proper CA (e.g. a company CA).
+{{% /note %}}  
+
+
+## All-in-one Certificates Setup
+For an all-in-one Cloudify manager, just the host's public key certificate is needed alongside its associated CA certificate and private key.
+
+
+## Replacing certificates
+Replacement of the certificates may be required as a result of regulatory compliance demand, certificate expiration, or revocation due to security breach.
+Follow the procedure described in the [Replacing Certificates guide](https://docs.cloudify.co/latest/cli/maint_cli/certificates/#replacing-certificates) when certificates' replacement is required.
```

Comment:
when using internal links, it is better to use a relative URL format to avoid cases where the links point to the wrong version. the format is:
[Replacing Certificates guide]({{< relref "cli/maint_cli/certificates/#replacing-certificates" >}})

Reply:
Thanks! I'll change it in all places.